docs/source/upcoming_release_notes/1318-TwinCATStatePositionerResetIndicator.rst

```diff
@@ -0,0 +1,30 @@
+1318 TwinCATStatePositionerResetIndicator
+#################
+
+API Breaks
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+- N/A
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- reset_cmd is now a PytmcSignal with io='io' instead of an EpicsSignal
```

Comment:
Sorry, maybe I'm at least a little picky with respect to context. This won't make any sense in the combined release notes. I'm expanding it slightly so that I understand where it came from later.
```suggestion
- TwinCATStatePositioner.reset_cmd is now a PytmcSignal with io='io' instead of an EpicsSignal to properly display the RBV readback PV in typhos.
```